packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -283,125 +283,137 @@ describe('KeyringController', () => {
   });
 
   describe('importAccountWithStrategy', () => {
-    it('should import account with strategy privateKey', async () => {
-      await expect(
-        keyringController.importAccountWithStrategy(
-          AccountImportStrategy.privateKey,
-          [],
-        ),
-      ).rejects.toThrow('Cannot import an empty key.');
```

Comment:
It looks like these two cases (empty key and invalid length) are no longer covered by tests

Reply:
Ah I see; they were hidden from the diff because they were duplicated before, now they are only tested once. Thanks, makes sense.